frontend/src/features/projects/assess/components/ReviewApproveActions.tsx

```diff
@@ -108,9 +114,21 @@ export const ReviewApproveActions = ({
             handleCancel={() => setDenyERP(false)}
             title="Deny Approval"
             message={
-              !values.exemptionRequested
-                ? 'Are you sure you want to deny the project for Enhanced Referral Process? Please ensure to provide reasoning in the shared notes prior to clicking deny.'
-                : 'Are you sure you want to deny this project with the request for exemption? Please ensure to provide reasoning in the shared notes prior to clicking deny.'
+              <>
+                {!values.exemptionRequested
+                  ? `Are you sure you want to deny the project for Enhanced
+                Referral Process? Please ensure to provide reasoning in the shared notes prior to
+                clicking deny.`
+                  : `Are you sure you want to deny this project with the request for
+                exemption? Please ensure to provide reasoning in the shared notes prior to clicking
+                deny.`}
+                {hasSubdivisions && (
```

Comment:
when denying, display the subdivision warning in the original warning message. We could decide to display this after the deny if we wanted, I just wanted to keep the # of clicks down.